src/inference/rejection.js

```diff
@@ -22,6 +21,17 @@ module.exports = function(env) {
       maxScore: 0,
       incremental: false
     });
+
+    this.throwOnError = options.throwOnError !== undefined ? options.throwOnError : true;
+    // the value of options.probe is the min sample rate
+    this.probe = options.probe;
```

Comment:
For enumeration, the probe means three things: a) estimate the complexity and quit if it's above threshold, b) quit when time out, c) `throwOnError: false`

So what are we going to do with a) and b), using two additional options like `maxComplexity: 200000` and `timeOut: 5000`?